src/scenes/EsiJaPerusopetus/lomake/7-MuutEhdot.js

```diff
@@ -23,24 +22,11 @@ const MuutEhdot = ({
 
   const onAddButtonClick = useCallback(
     payload => {
-      actions.createTextBoxChangeObject(sectionId);
+      actions.addAClick(sectionId, payload.anchor);
```

Comment:
Tässä pitäisi käyttää createTextBoxChangeObjectsia?

Reply:
Näin on.